transform/mattermost-analytics/models/staging/cws/_cws__models.yml

```diff
@@ -64,3 +64,23 @@ models:
       - name: is_terms_accepted
         description: Whether the user accepted the terms and conditions.
 
+  - name: stg_cws__marketing
+    description: |
+      Security Newsletter subscription data.
```

Comment:
based on the content of the `stg_cws__marketing.sql` file (there is nothing security-newsletter specific in the code fwis), there is no need to use "Security Newsletter" in the description, we can keep it generic (it just happens that the current content is only security newsletter related). Would also remove "Newsletter" from the `created_at` and `updated_at` fields.

Reply:
The descriptions are now updated for staging table.